libr/util/strbuf.c

```diff
@@ -141,6 +154,16 @@ R_API char *r_strbuf_get(RStrBuf *sb) {
 	return sb? (sb->ptr? sb->ptr: sb->buf) : NULL;
 }
 
+R_API ut8 *r_strbuf_getbin(RStrBuf *sb, int *len) {
+	if (sb) {
```

Comment:
We can work out this in a separate PR, i prefer to stay safe for now and its not really a huge performance issue, but i agree that we should define some design rules to avoid such checks on internal apis